data/json/items/generic.json

```
@@ -155,6 +155,38 @@
     "name": { "str": "ANOMALOUS ARTIFACT REPORT" },
     "snippet_category": "artifact_scp"
   },
+  {
+    "type": "GENERIC",
+    "id": "small_printout_portal_storm_data",
+    "name": { "str": "short NRE printout" },
+    "category": "books",
+    "description": "A sequence of overlapping numbers plotted in thermal paper, at times so so densely packed that they disappear into thick blots of oxide colored ink.  A few number sequences repeat in an almost orderly fashion, but you cant tell wether that means anything.",
+    "weight": "1 g",
+    "to_hit": -3,
+    "color": "white",
+    "symbol": ",",
+    "material": [ "paper" ],
+    "volume": "50 ml",
+    "flags": [ "TRADER_AVOID" ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "medium_printout_portal_storm_data",
+    "copy-from": "small_printout_portal_storm_data",
+    "name": { "str": "NRE printout" },
+    "description": "A long sequence of overlapping numbers plotted in thermal paper, at times so densely packed that they disappear into thick blots of oxide colored ink.  Blots and numbers appear to be arranged in a strangely pleasing pattern.",
+    "weight": "3 g",
+    "volume": "4 ml"
```

```suggestion
    "volume": "50 ml"
```
I'm assuming this is a typo?

---

Not really its a tiny piece of receipt paper.

---

I'm just a bit confused, the small printout is 50ml, medium 4ml, large 50ml